assets/css/style.css

```diff
@@ -0,0 +1,31 @@
+/*
+Don't underline hyperlinks
+*/
+a {
```

Comment:
Please look into a more careful solution that uses additional selectors suggested in https://stackoverflow.com/a/69299851/5775947